### PR TITLE
[site] Added how to implement Deckhouse page

### PR DESCRIPTION
**Pull request by ToyTox**

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/pages/how_to_buy_ru.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+<section class="block deckhouse-in-register">
+  <div class="container">
+    <h2 class="text text_lead" id="russian-k8s">
+      Первое знакомство
+    </h2>
+    <div class="block__columns block__columns_top">
+      <div>
+        <p class="text text_big">
+          У нас есть бесплатная версия платформы — <a href="https://deckhouse.ru/community/community_edition.html">Deckhouse&nbsp;CE</a>. Чтобы проверить ее&nbsp;в&nbsp;действии, воспользуйтесь <a href="https://deckhouse.ru/gs/">руководством по&nbsp;быстрому старту</a>. Если что-то не&nbsp;получается, обращайтесь за&nbsp;помощью в&nbsp;наш&nbsp;<a href="https://t.me/deckhouse_ru">Telegram-канал</a>.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:39]:
```suggestion
          У нас есть бесплатная версия платформы — <a href="community/community_edition.html">Deckhouse&nbsp;CE</a>. Чтобы проверить ее&nbsp;в&nbsp;действии, воспользуйтесь <a href="gs/">руководством по&nbsp;быстрому старту</a>. Если что-то не&nbsp;получается, обращайтесь за&nbsp;помощью в&nbsp;наш&nbsp;<a href="https://t.me/deckhouse_ru">Telegram-канал</a>.
```

File: docs/site/pages/how_to_buy_ru.html [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        </p>
+      </div>
+      <div>
+        <p class="text text_big">
+          Чтобы протестировать полнофункциональную версию, <a href="https://deckhouse.ru/products/enterprise_edition.html">запросите</a> триальный ключ доступа, который действует в&nbsp;течение 30 дней. Об&nbsp;отличиях бесплатной и&nbsp;платной версий платформы читайте в&nbsp;разделе <a href="https://deckhouse.ru/community/community_edition.html#ce-vs-ee">«Deckhouse Platform&nbsp;CE vs. EE»</a>.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:44]:
```suggestion
          Чтобы протестировать полнофункциональную версию, <a href="products/enterprise_edition.html">запросите</a> триальный ключ доступа, который действует в&nbsp;течение 30 дней. Об&nbsp;отличиях бесплатной и&nbsp;платной версий платформы читайте в&nbsp;разделе <a href="community/community_edition.html#ce-vs-ee">«Deckhouse Platform&nbsp;CE vs. EE»</a>.
```

File: docs/site/pages/how_to_buy_ru.html [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+          Варианты лицензирования и&nbsp;поддержки
+        </h2>
+      </div>
+      <ul class="features__item-list">
+        <p>
+          На встречах детально обсуждается планируемая архитектура и&nbsp;необходимое количество ресурсов. Для подготовки полноценного КП нам нужно понимать:
+        </p>
+        <li>
+          <strong>Требуемое количество и&nbsp;тип лицензий</strong>.<br/> Подробнее о&nbsp;доступных вариантах лицензирования читайте
+          <a href="https://deckhouse.ru/license_rules.html">в&nbsp;«Правилах лицензирования»</a>.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:163]:
```suggestion
          <a href="license_rules.html">в&nbsp;«Правилах лицензирования»</a>.
```

File: docs/site/pages/how_to_buy_ru.html [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+        </li>
+        <li>
+          <strong>Необходимый уровень и&nbsp;объем технической поддержки</strong>.<br> Подробнее о&nbsp;параметрах технической поддержки читайте
+          <a href="https://deckhouse.ru/regulations.html">в&nbsp;«Регламенте технической поддержки»</a>.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:167]:
```suggestion
          <a href="regulations.html">в&nbsp;«Регламенте технической поддержки»</a>.
```

File: docs/site/pages/how_to_buy_ru.html [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        </li>
+        <p>
+          Мы можем <a href="https://deckhouse.ru/services/training.html">обучить</a> ваших специалистов работе с&nbsp;платформой и/или заложить в&nbsp;договор необходимый объем консультаций для совместной проработки архитектурных вопросов.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:170]:
```suggestion
          Мы можем <a href="services/training.html">обучить</a> ваших специалистов работе с&nbsp;платформой и/или заложить в&nbsp;договор необходимый объем консультаций для совместной проработки архитектурных вопросов.
```

File: docs/site/pages/how_to_buy_ru.html [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+  <div class="features__item features__item_even">
+    <div class="features__item-content container">
+      <div class="features__item-header">
+        <h2 class="features__item-title text_h1">
+          Помощь вашей команде
+        </h2>
+      </div>
+      <ul class="features__item-list">
+        <li>
+          Чтобы снизить нагрузку на&nbsp;вашу команду и&nbsp;сократить сроки внедрения платформы, вы&nbsp;можете поручить эту работу нашим инженерам. В&nbsp;рамках услуги <a href="https://deckhouse.ru/services/managed.html">Managed Deckhouse</a> мы&nbsp;сами развернем кластеры и&nbsp;обеспечим их&nbsp;дальнейшую поддержку 24х7.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:187]:
```suggestion
          Чтобы снизить нагрузку на&nbsp;вашу команду и&nbsp;сократить сроки внедрения платформы, вы&nbsp;можете поручить эту работу нашим инженерам. В&nbsp;рамках услуги <a href="services/managed.html">Managed Deckhouse</a> мы&nbsp;сами развернем кластеры и&nbsp;обеспечим их&nbsp;дальнейшую поддержку 24х7.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/pages/how_to_buy_ru.html [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+          Свяжитесь с&nbsp;нами любым удобным способом:
+          <ul>
+            <li>
+              По телефону +7 (495) 721-10-27.
+            </li>
+            <li>
+              По почте <a href="mailto:sales@deckhouse.io">sales@deckhouse.io</a>.
+            </li>
+            <li>
+              Через онлайн-чат на&nbsp;сайте deckhouse.ru.
```
Comment (z9r5) [inline docs/site/pages/how_to_buy_ru.html:70]:
```suggestion
              Через онлайн-чат.
```

**Review by z9r5 — APPROVED** [review]

**Pull request merged**